geometry/distance_on_sphere.cpp

```diff
@@ -47,9 +32,65 @@ double DistanceOnEarth(LatLon const & ll1, LatLon const & ll2)
   return DistanceOnEarth(ll1.m_lat, ll1.m_lon, ll2.m_lat, ll2.m_lon);
 }
 
+m3::PointD GetPointOnSphere(ms::LatLon const & ll, double sphereRadius)
+{
+  ASSERT(LatLon::kMinLat <= ll.m_lat && ll.m_lat <= LatLon::kMaxLat, ());
+  ASSERT(LatLon::kMinLon <= ll.m_lon && ll.m_lon <= LatLon::kMaxLon, ());
+
+  // The point (lat=0, lon=+90°) translates to (x=1, y=0, z=0).
+  // The point (lat=0, lon=0)   translates to (x=0, y=1, z=0).
+  // The point (lat=+90°, lon=0) translates to (x=0, y=0, z=1).
```

Comment:
1. This comment looks wrong now
2. Move it to hpp?

Reply:
why it should be in hpp? It's implementation details

Reply:
Why should it be anywhere at all then?
If it is important how the axes are located, make the user of the API informed.